pkg/utils/parse_arf_result.go

```diff
@@ -389,11 +400,19 @@ func remediationFromFixElement(scheme *runtime.Scheme, fix *xmlquery.Node, scanN
 	dnsFriendlyFixId := strings.ReplaceAll(fixId, "_", "-")
 	remName := fmt.Sprintf("%s-%s", scanName, dnsFriendlyFixId)
 	// TODO(OZZ) fix text
-	return remediationsFromString(scheme, remName, namespace, fix.InnerText())
+	return remediationsFromString(scheme, remName, namespace, fix.InnerText(), resultValues)
 }
 
-func remediationsFromString(scheme *runtime.Scheme, name string, namespace string, fixContent string) []*compv1alpha1.ComplianceRemediation {
-	objs, err := ReadObjectsFromYAML(strings.NewReader(fixContent))
+func remediationsFromString(scheme *runtime.Scheme, name string, namespace string, fixContent string, resultValues map[string]string) []*compv1alpha1.ComplianceRemediation {
+	//ToDO find and substitute the value
+	fixWithValue, valuesUsedList, notFoundValueList, parsingError := parseValues(fixContent, resultValues)
+
+	if (len(notFoundValueList) == 0) && (len(valuesUsedList) == 0) && parsingError != nil {
```

Comment:
the braces seem to be extra since you're and-ing three conditions, but I wonder if there's a condition where you want to continue if parsingError is nil?

Reply:
When the templating is not used then the parsingError will be nil, and It will continue, did I understand your question correctly?